GitCommands/Git/GitModule.cs

```diff
@@ -4146,24 +4135,11 @@ public bool StopTrackingFile(string filename)
                 "--abbrev=40",
                 commitId
             };
-            var output = _gitExecutable.GetOutput(args).TrimEnd();
-
-            if (IsGitErrorMessage(output))
-            {
-                return null;
-            }
-
-            return output;
-        }
 
-        /// <summary>
-        /// Determines whether a git command's output indicates an error occurred.
-        /// </summary>
-        /// <param name="gitOutput">The output from the git command, to inspect.</param>
-        /// <returns><c>true</c> if the command detailed an error, otherwise <c>false</c>.</returns>
-        public static bool IsGitErrorMessage(string gitOutput)
-        {
-            return Regex.IsMatch(gitOutput, @"^\s*(error:|fatal)");
+            ExecutionResult exec = _gitExecutable.Execute(args);
+            return !exec.ExitedSuccessfully
+                ? exec.StandardOutput.TrimEnd()
+                : null;
```

Comment:
ditto

Reply:
Exit > 0 is normal if no tag exists where the command was started, must not raise errors